src/jsx-runtime.ts

```diff
@@ -25,7 +25,7 @@ export const useMDXComponents = (): Record<string, (properties: ParentProps) =>
 const REPLACED_COMPAT_SET = new Set(['mjx']);
 const compatRegExp = new RegExp(`(?:${[...REPLACED_COMPAT_SET].join('|')})-.+`, 'g');
 
-const expressionCache: Record<string, string> = {};
+const expressionCache: Record<string, string> = Object.create(null);
```

Comment:
could also use `Map` but empty object simply just works

Reply:
Hey, @mdynnl. Thanks for this. This is just a cache for resolved regular expressions which result from attribute names conversion, i.e. strokeWidth -> stroke-width, and similar. So, I don't think this is a bug per se, but the PR itself is OK, and I'm going to merge it if https://github.com/high1/solid-mathjax works with it.

Reply:
![image](https://user-images.githubusercontent.com/783303/190852884-b61d7b27-a3a1-4190-a9c3-394f4080f770.png)
There's a typescript warning here, this returns any... Since there's no typed call here, would you suggest type casting with as like `const expressionCache = Object.create(null) as Record<string, string>;` or leaving the current code intact?